### PR TITLE
Update cli

### PR DESCRIPTION
**Pull request by BenT00le**
use api.algorithmia.com for default api address if non is provided in auth

### CODE REVIEW

File: Algorithmia/__main__.py [hunk 1]
```diff
@@ -110,20 +110,24 @@ def main():
...
+        if len(APIkey) == 28 and APIkey.startswith("sim"):
+            if APIaddress == "" or not APIaddress.startswith("https://api."):
+                APIaddress = "https://api.algorithmia.com"
+
             CLI().auth(APIkey, APIaddress, args.profile)
         else:
             print("invalid api key")
 
-    if(args.cmd == 'help'):
+    ifargs.cmd == 'help':
```
Comment (kennydaniel) [inline Algorithmia/__main__.py:125]: typo, missing space

File: out.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Hello test
```
Comment (kennydaniel) [inline out.txt:1]: Again with out.txt. please read your code diff before pushing. Also let's put this in gitignore.

**Comment by aslisabanci** [comment]
Hey @BenT00le , I reopened the issue regarding [my tests on deepPurple](https://github.com/algorithmiaio/algorithmia-python/issues/33). So I'll be holding off the approval until I can get that fix working :)

**Comment by aslisabanci** [comment]
I also have this issue: https://github.com/algorithmiaio/algorithmia-python/issues/35 but if you think this shouldn't block the release, we can plan to fix it for the next version.

**Pull request merged**